src/audiomixerboard.cpp

```diff
@@ -1267,7 +1267,7 @@ void CAudioMixerBoard::ApplyNewConClientList ( CVector<CChannelInfo>& vecChanInf
                 vecpChanFader[iChanID]->Reset();
                 vecAvgLevels[iChanID] = 0.0f;
 
-                if ( static_cast<int> ( iChanID ) == iMyChannelID )
+                if ( static_cast<int> ( iChanID ) == pClient->GetMyChannelID() )
```

Comment:
Maybe stick `pClient->GetMyChannelID()` into a variable as it's used in a few places here.

Reply:
The value doesn't change while connected, and it is just a simple Get method, so I don't think threads are relevant in this case. However, I see most of these uses are within a loop, so I'm happy to fetch once to a local variable within each function and use that within the loop. I'll add a commit for that shortly.

Reply:
Done 072b2af